build.gradle

```
@@ -1,22 +1,22 @@
 buildscript {
     ext {
         compose = '1.7.0-alpha08'
-        composeActivity = '1.9.0'
+        composeActivity = '1.9.2'
 
         ktxCore = '1.12.0'
         ktxLifeCycle = '2.6.2'
 
         material = '1.6.7'
```

We must also update the material to 1.7.2 to see the issue. 1.6.7 does not have it.

```suggestion
        material = '1.7.2'
```

---

just added compose bom :D